### PR TITLE
Add Portkey's perpetual free tier in a new category "Generative AI"

### PR DESCRIPTION
**Pull request by vrushankportkey**
* [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1051,6 +1052,12 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
 **[⬆️ Back to Top](#table-of-contents)**
 
+## Generative AI
+
+* [Portkey](https://portkey.ai/) - Control panel for Gen AI apps featuring an observability suite & an AI gateway. Send & log up to 10,000 requests for free every month.
```
Comment (ripienaar) [inline README.md:1057]: Thanks, we dont add single entry categories.

Please move other AI things in here or add your entry to an existing category.

Comment (vrushankportkey) [reply README.md:1057]: @ripienaar added a couple more tools in the category! Would that work?

Comment (ripienaar) [reply README.md:1057]: yes, unfortunately gradient doesnt list pricing nor do they specifically say the standard plan is free so I cant add it

Comment (vrushankportkey) [reply README.md:1057]: Oh shoot you're right. Sorry about that. Removed that, and added another tool. Hope that works.

**Comment by ripienaar** [comment]
thank you

**Comment by imba-tjd** [comment]
perhaps openpipe changed its free tier?
https://openpipe.ai/pricing
they offer 100$ credits. but I didn't see *always free* plan.
Shall we change the description or remove it from the list?

**Comment by ripienaar** [comment]
Yeah I checked it when this was added but maybe its changed, we should remove it now.

**Pull request merged**